### PR TITLE
Install the support requests using PowerShell 

### PR DESCRIPTION
**Pull request by joelverhagen**
Today, the installation is manual. Gross!

Also removed unused code from the CopyAzureContainer PS1.

### CODE REVIEW

**Comment by joelverhagen** [comment]
Address https://github.com/NuGet/Engineering/issues/2099

File: src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    Install-NuGetScheduledTask $STTrigger "Support Requests Weekly Notification" "WeeklySummaryNotification.cmd"
+}
+
+Function Install-NuGetScheduledTask
+{
+    param($STTrigger, $STName, $cmd)
+
+    #Action to run as
+    $cmdexe = [system.environment]::getenvironmentvariable("ComSpec")
```
Comment (agr) [inline src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1]: Capitalization, please

**Comment by agr** [comment]
Does it behave as expected when run several times?

**Comment by joelverhagen** [comment]
> Does it behave as expected when run several times?

Yes, it replaces the scheduled task name.

File: src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1 (hunk 1 above)
Comment (joelverhagen) [reply src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1]: Sure. This was copied from another script but I will fix it here.

File: src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1 [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Function Install-WeeklyTask
+{
+    $STTrigger = New-ScheduledTaskTrigger -Weekly -WeeksInterval 1 -DaysOfWeek Monday -At "12pm"
+
+    Install-NuGetScheduledTask $STTrigger "Support Requests Weekly Notification" "WeeklySummaryNotification.cmd"
+}
+
+Function Install-NuGetScheduledTask
+{
+    param($STTrigger, $STName, $cmd)
```
Comment (agr) [inline src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1]: More appropriate argument names? `Trigger`, `Name`, `Command`?

File: src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1 [hunk 3]
```diff
@@ -0,0 +1,31 @@
+Function Install-DailyTask
+{
+    $STTrigger = New-ScheduledTaskTrigger -DaysInterval 1 -At "12pm" -Daily
+
+    Install-NuGetScheduledTask $STTrigger "Support Request Daily Notification" "OnCallDailyNotification.cmd"
```
Comment (agr) [inline src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1]: Specify argument names?

Comment (joelverhagen) [reply src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1]: Sure!

File: src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1 (hunk 2 above)
Comment (joelverhagen) [reply src/NuGet.SupportRequests.Notifications/Scripts/Functions.ps1]: Sure!

File: src/NuGet.SupportRequests.Notifications/NuGet.SupportRequests.Notifications.csproj [hunk 4]
```diff
@@ -123,5 +123,10 @@
   </PropertyGroup>
   <Import Project="$(SignPath)\sign.targets" Condition="Exists('$(SignPath)\sign.targets')" />
   <Import Project="$(SignPath)\sign.microbuild.targets" Condition="Exists('$(SignPath)\sign.microbuild.targets')" />
+  <ItemGroup>
+    <PowerShellScriptsToNotSign Include="PreDeploy.ps1"/>
```
Comment (loic-sharma) [inline src/NuGet.SupportRequests.Notifications/NuGet.SupportRequests.Notifications.csproj]: Out of curiosity, why don't we want to sign this script?

Comment (joelverhagen) [reply src/NuGet.SupportRequests.Notifications/NuGet.SupportRequests.Notifications.csproj]: It doesn't exist in this project.

**Pull request merged**